docs/develop/node/intro/types-of-node.md

```diff
@@ -0,0 +1,16 @@
+---
+id: types-of-node
+title: Types of Node
+sidebar_label: Types of Node
+---
+
+You can run three different types of node - Validator Node, RPC Node, and Archival Node.
+
+## Validator Node
+Validator Nodes are the operators of the NEAR blockchain and are essential to the health of the network. Validator nodes receive new states and blocks while participating in block validation, and verify blocks and states.
+
+## RPC Node
+RPC Nodes are RPC service providers for that provide public RPC endpoints for developers to use. The NEAR Inc currently maintains a public RPC endpoint `http://rpc.mainnet.near.org/` that is free for everyone to use. However, any participants are encouraged to run their own RPC node and offer RPC services through a grant from the NEAR Foundation. For more information, please reach out in Validator channels on Telegram or Discord.
```

Comment:
The rpc endpoint is technically provided by NEAR Foundation I believe. cc @eriktrautman @ilblackdragon